README.md

```diff
@@ -633,6 +636,7 @@ XestiMonitors is available under [the MIT license][license].
 [focus_monitor]:                                https://eBardX.github.io/XestiMonitors/Classes/FocusMonitor.html
 [gyroscope_monitor]:                            https://eBardX.github.io/XestiMonitors/Classes/GyroscopeMonitor.html
 [heading_monitor]:                              https://eBardX.github.io/XestiMonitors/Classes/HeadingMonitor.html
+[http_cookie_storage_monitor]:          https://eBardX.github.io/XestiMonitors/Classes/HTTPCookieStorageMonitor.html
```

Comment:
Please also update lines 20 and 21.

Reply:
I am a bit lost here. Is that in the readMe file?

Reply:
My mistake. I meant lines 20 and 21 of `Documents/wrapped-foundation-notifications.md`.

Reply:
Don't worry about it. I will make the final fixes and merge this in.